migrate.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+repo_url="https://github.com/vsc-eco/vsc-deployment.git"
+default_path=$(realpath "../vsc-deployment")
+
+if [ -z "$1" ]; then
+    echo "VSC node migration script. The default path is for the deployment repository is '${default_path}'"
+    read -p "Enter the path to clone the repository [${default_path}]: " clone_path
+    clone_path="${clone_path:-$default_path}"
+else
+    clone_path="$1"
+fi
+
+
+
+git clone "$repo_url" "$clone_path" || { echo "Failed to clone repository."; exit 1; }
+
+# Check if the .env file and data directory do not exist in the target and do exist in the current directory
+if [ ! -f "${clone_path}/.env" ] && [ ! -d "${clone_path}/data" ] && [ -f ".env" ] && [ -d "data" ]; then
+    cp ".env" "${clone_path}/"
+    cp -r "data" "${clone_path}/"
+    echo "Node data copied successfully."
```

Comment:
Wouldn't this work with your other PR https://github.com/vsc-eco/vsc-deployment/pull/5?
```suggestion
    echo "Node data copied successfully."

    read -p "Would you like to enable automatic updates? [y]: " -n 1 -e auto_updates
    if [ "y" = "${auto_updates:-y}" ]; then
        echo "AUTO_UPDATE=true" >> "${clone_path}/.env"
    else
        echo "AUTO_UPDATE=false" >> "${clone_path}/.env"
    fi
```

Reply:
oh yeah sure that would work :)

but on the other side we now present the users with the option and then they may tend to disable the auto update

if they dont specify the variable, auto update is automatically enabled (variable not set -> enabled)

would you prefer asking the users or would you prefer to see auto updates as "manually opting out", for which they would need to locate the disabling variable in the description of the new deployment

Reply:
on the flipside people may not solely use the old deployment because they dont know about the new deployment, but because the old one didnt force auto updates upon them, so maybe we shouldnt annoy those kinds of people and just ask them..

Reply:
Yeah, you're right. We want to discourage them from opting out, but we don't want to hide it either. Maybe we can add a warning that if they don't promptly update their node, they may be excluded from the network.

Reply:
okay thats good middle ground, lets do it that way 🤝

Reply:
added a commit